ADALiOS/ADALiOS/ADAuthenticationContext.m

```diff
@@ -135,6 +135,7 @@ - (id)initWithAuthority:(NSString*) authority
         _authority = extractedAuthority;
         _validateAuthority = bValidate;
         _tokenCacheStore = tokenCache;
+        _credentialsType = AD_CREDENTIALS_AUTO;
```

Comment:
Make the default AD_CREDENTIALS_EMBEDDED, we want to make the broker an opt-in thing as well.